react-pr2/src/components/Header/index.jsx

```diff
@@ -0,0 +1,53 @@
+import { AppBar, Box, Button, Container, Toolbar, Typography } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import Filter from './Filters/index';
+
+const useStyles = makeStyles({
+  root: {},
+  logoIcon: {
+    marginRight: "100px",
+  },
+  header: {
+    height: "100px"
+  },
+  account: {
+    display: "flex",
+    alignItemsL: "center",
+    gap: "3px"
+  }
+});
+
+export default function Header() {
+  const classes = useStyles();
+
+  return (
+    <Box>
+      <AppBar position="static">
```

Comment:
```suggestion
      <AppBar>
```
Mặc định là static rồi mà nhỉ